src/forms-opportunities/utils.js

```diff
@@ -182,6 +182,14 @@ export async function generateOpptyDataForHighPageViewsLowFormNav(formVitals, co
   );
 }
 
+export function isSearchForm(scrapedFormData) {
```

Comment:
unsubscribe is not a search form. I am assuming we are creating a single API  to identify `isRelevantForm` rather than having isSearchForm, isFilterForm, isCalculatorForm, ...etc.

Reply:
yeah agree, we can do that as an improvement.